vllm/platforms/cuda.py

```diff
@@ -243,6 +257,16 @@ def get_attn_backend_cls(cls, selected_backend, head_size, dtype,
                 f"with use_v1: {use_v1} use_mla: {use_mla}")
 
         target_backend = _Backend.FLASH_ATTN
+        if cls.has_device_capability(100):
```

Comment:
seems we will never use FLASH_ATTN for Blackwell, is this expected?

Reply:
I messed up my state here, will update